src/VisualStudio/Core/Impl/CodeModel/CodeModelEventQueue.cs

```diff
@@ -35,6 +35,7 @@ private void EnqueueEvent(CodeModelEvent @event)
             {
                 var priorEvent = _eventQueue.Peek();
                 if (priorEvent.Node == @event.Node &&
+                    priorEvent.ParentNode == @event.ParentNode &&
```

Comment:
This shouldn't be necessary.  By definition if a node is == (ReferenceEqual), the parent must be the same.